SETUP.md

```diff
@@ -137,6 +147,16 @@ sed --in-place "s/manubot-rootstock/$REPO/g" README.md
 git rm content/02.delete-me.md
 ```
 
+For setup on MAC OSX using BSD sed use the following commands.
+
+```sh
+sed -i '' "s/greenelab/$OWNER/g" README.md
```

Comment:
What if we abandoned `--inplace`/`-1` and just ran a subseqent `mv` command?

Reply:
We could have the first sed command output to tmp file and the send one take tmp and output into README.md, then follow with rm for tmp file.

Reply:
If you use `mv -f`, you can skip the `rm` command.